.github/PULL_REQUEST_TEMPLATE.md

```
@@ -0,0 +1,18 @@
+## خلاصه تغییرات
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)